File: typescript/exercices/corrections/tp-final/classes/customer.ts
import { BaseClass } from "./base"

export class Customer extends BaseClass {
   
    firstName:string
    lastName:string
    phone:string
    constructor(id:number, firstName:string, lastName:string, phone:string) {
        super(id)
        this.firstName = firstName
        this.lastName = lastName
        this.phone = phone
    }
}

File: FormationDotNet/demoEntityFrameworkDbFirst/Classes/Customer.cs
using System;
using System.Collections.Generic;

namespace demoEntityFrameworkDbFirst.Classes
{
    public partial class Customer
    {
        public Customer()
        {
            Accounts = new HashSet<Account>();
        }

        public int Id { get; set; }
        public string Phone { get; set; } = null!;
        public string FirstName { get; set; } = null!;
        public string LastName { get; set; } = null!;

        public virtual ICollection<Account> Accounts { get; set; }
    }
}
